libtenzir/builtins/operators/diagnostics.cpp

```diff
@@ -0,0 +1,53 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2024 The Tenzir Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <tenzir/argument_parser.hpp>
+#include <tenzir/plugin.hpp>
+#include <tenzir/series_builder.hpp>
+
+#include <caf/typed_event_based_actor.hpp>
+
+namespace tenzir::plugins::diagnostics {
+
+namespace {
+
+class plugin final : public virtual operator_parser_plugin {
+public:
+  auto name() const -> std::string override {
+    return "diagnostics";
+  };
+
+  auto signature() const -> operator_signature override {
+    return {.source = true};
+  }
+
+  auto parse_operator(parser_interface& p) const -> operator_ptr override {
+    auto parser = argument_parser{"diagnostics", "https://docs.tenzir.com/next/"
+                                                 "operators/diagnostics"};
+    bool live = false;
+    parser.add("--live", live);
+    parser.parse(p);
+    const auto definition = fmt::format("export --internal {}| where #schema "
+                                        "== /tenzir\\.diagnostic\\.*/",
+                                        live ? " --live" : "");
```

Comment:
```suggestion
    const auto definition = fmt::format("export --internal {}| where #schema "
                                        "== \"tenzir.diagnostic\"",
                                        live ? " --live" : "");
```

Reply:
Changed in https://github.com/tenzir/tenzir/pull/3828/commits/e25463acad78daf8b7adca97225cc3cbaa3fa14f